recipes/construct/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "construct" %}
+{% set version = "2.9.45" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2271a0efd0798679dea825ff47e22a4c550456a5db0ba8baa82f7eae0af0118c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - construct
+    - construct.lib
+
+about:
+  home: http://construct.readthedocs.org
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: A powerful declarative symmetric parser/builder for binary data
+  doc_url: 
+  dev_url: 
```

Comment:
These line scan be either filled in or removed.

Reply:
I opted to remove them.